src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt

```diff
@@ -90,6 +92,10 @@ class ChristmasEventPlugin : JavaPlugin() {
         PacketEvents.getAPI().init()
     }
 
+    private fun registerDonations() {
+        DonationListener(config.getString("donations.campaignId")?: "", config.getString("donations.authToken")?: "") //TODO: Change this up however you want
```

Comment:
I'd much rather have an error being thrown here than have an error be thrown inside DonationListener.init